user_docs/en/changes.t2t

```diff
@@ -7,12 +7,12 @@ What's New in NVDA
 This release includes optional experimental support for UIA in Excel and Chromium browsers.
 There are fixes for several languages, and for accessing links in Braille.
 There are updates to Unicode CLDR and mathematical symbols, eSpeak-NG and LibLouis.
-
 Many bug fixes and improvements, including in Office, Visual Studio and several languages.
-Note: this is an Add-on API compatibility breaking release.
-Add-ons will need to be re-tested and have their manifest updated.
 This release also drops support for Adobe Flash.

```

Comment:
I think the intention here was to make this an item of the "Note", perhaps @Qchristensen can clarify?

Reply:
Could we bolden this text and maybe say "Warning" instead. The increase in urgency might decrease the number of people who miss/ignore it.

Reply:
Looking at the commits in the initial PR (#12376), we can see that all the sentences of the blurb were in the initial paragraph. Then the paragraph was split in two, probably unintentionally when changing "mathematic" to "mathematical".
Putting all the sentences in the first paragraph and the note regarding add-ons in a second one was my initiative in this PR only (not yet merged).